legend-engine-language-pure-store-relational/src/main/java/org/finos/legend/engine/language/pure/compiler/toPureGraph/HelperRelationalDatabaseConnectionBuilder.java

```diff
@@ -50,13 +50,16 @@ public static void addDatabaseConnectionProperties(org.finos.legend.pure.m3.core
         try
         {
             connection._element(HelperRelationalBuilder.resolveDatabase(element, elementSourceInformation, context));
-            connection._timeZone(timeZone);
-            connection._quoteIdentifiers(quoteIdentifiers);
         }
         catch (RuntimeException e)
         {
             connection._element(new Root_meta_relational_metamodel_Database_Impl(element)._name(element));
         }
+        finally
```

Comment:
I guess this is fine too, but why don't we just move all of these to before the `try ... catch` block instead of introducing this `finally` block?

Reply:
indeed!

Reply:
yes this makes sense